src/main/java/com/ning/http/client/providers/grizzly/GrizzlyResponseBodyPart.java

```diff
@@ -35,7 +35,7 @@
 public class GrizzlyResponseBodyPart extends HttpResponseBodyPart {
 
     private final HttpContent content;
-    private final Connection connection;
+    private final FilterChainContext context;
```

Comment:
Wow, was no one really using the connection? I don't see any other replacement with `context.getConnection()`

Reply:
Nope, and as it was a private field with no getter, it can't be used from an eventual child implementation